stores/custom-roles/README.md

```diff
@@ -1,7 +1,208 @@
-# OpenFGA Sample Store
+# OpenFGA Custom Roles Sample Store
 
 * **Title**: **Custom Roles** 
 * **Documentation**: https://openfga.dev/docs/modeling/custom-roles
 * **Playground**: https://play.fga.dev/sandbox/?store=custom-roles
 
+## Table of Content
```

Comment:
I learned yesterday that ToCs are autogenerated by GitHub. https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/

Reply:
They do generate them, but they're not as visible, so a lot of people miss out on them :(